app/assets/js/rtMedia.backbone.js

```diff
@@ -497,7 +497,14 @@ jQuery( function( $ ) {
 			},
 			initialize: function( config ) {
 				this.uploader = new plupload.Uploader( config );
-				/*
+				
+				//current_url will fetch present URL
+				var current_url = document.URL;
+				if( -1 !== current_url.indexOf( 'message' ) ){
```

Comment:
Change it to `if ( -1 !== current_url.indexOf( 'message' ) ) {`

Please follow WordPress JS coding standards to keep code consistent.